--- doc/sources/conf.py.orig	2023-05-20 19:06:18
+++ doc/sources/conf.py	2023-05-20 19:06:25
@@ -32,8 +32,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
     'autodoc', 'sphinx.ext.todo', 'preprocess', 'sphinx.ext.ifconfig',
-    'sphinx.ext.viewcode', 'sphinx.ext.mathjax', 'sphinx.ext.extlinks',
-    'sphinxcontrib.jquery']
+    'sphinx.ext.viewcode', 'sphinx.ext.mathjax', 'sphinx.ext.extlinks']
 
 if sphinx.version_info[0] >= 4:
     # In 4.0 and above has been added the support to substitute by ‘%s’ in the caption.
